content/deploy/restore/influxdb.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: 'Restore InfluxDB'
 date: '2023-10-19T13:52:23-03:00'
```

Comment:
Can we set to this is a draft until we have updated the installation to include influxdb?